public/app/features/dashboard/dashgrid/DashboardEmpty/DashboardEmpty.tsx

```diff
@@ -28,100 +29,113 @@ interface InternalProps {
-          <Stack direction={{ xs: 'column', md: 'row' }} wrap="wrap" gap={4}>
-            <Box borderRadius="lg" borderColor="strong" borderStyle="dashed" padding={3} flex={1}>
-              <Stack direction="column" alignItems="center" gap={1}>
-                <Text element="h3" textAlignment="center" weight="medium">
-                  <Trans i18nKey="dashboard.empty.add-library-panel-header">Import panel</Trans>
-                </Text>
-                <Box marginBottom={2}>
+                <Box marginBottom={2} paddingX={4}>
                   <Text element="p" textAlignment="center" color="secondary">
-                    <Trans i18nKey="dashboard.empty.add-library-panel-body">
-                      Add visualizations that are shared with other dashboards.
+                    <Trans i18nKey="dashboard.empty.add-visualization-body">
+                      Select a data source and then query and visualize your data with charts, stats and tables or
+                      create lists, markdowns and other widgets.
                     </Trans>
                   </Text>
                 </Box>
                 <Button
+                  size="lg"
                   icon="plus"
-                  fill="outline"
-                  data-testid={selectors.pages.AddDashboard.itemButton('Add a panel from the panel library button')}
-                  onClick={onAddLibraryPanel}
-                  disabled={!onAddLibraryPanel}
+                  data-testid={selectors.pages.AddDashboard.itemButton('Create new panel button')}
+                  onClick={onAddVisualization}
+                  disabled={!onAddVisualization}
                 >
-                  <Trans i18nKey="dashboard.empty.add-library-panel-button">Add library panel</Trans>
+                  <Trans i18nKey="dashboard.empty.add-visualization-button">Add visualization</Trans>
                 </Button>
               </Stack>
             </Box>
-            <Box borderRadius="lg" borderColor="strong" borderStyle="dashed" padding={3} flex={1}>
-              <Stack direction="column" alignItems="center" gap={1}>
-                <Text element="h3" textAlignment="center" weight="medium">
-                  <Trans i18nKey="dashboard.empty.import-a-dashboard-header">Import a dashboard</Trans>
-                </Text>
-                <Box marginBottom={2}>
-                  <Text element="p" textAlignment="center" color="secondary">
-                    <Trans i18nKey="dashboard.empty.import-a-dashboard-body">
-                      Import dashboards from files or{' '}
-                      <TextLink external href="https://grafana.com/grafana/dashboards/">
-                        grafana.com
-                      </TextLink>
-                      .
-                    </Trans>
+
+            {/* Suggested Dashboards Section */}
+            {config.featureToggles.suggestedDashboards &&
+              !config.featureToggles.dashboardLibrary &&
```

Comment:
I think this detail would be key to highlight in the PR description in all caps 😬 

e.g. to try out / or enable this feature you must TURN ON `suggestedDashboards` feature toggle and TURN OFF `dashboardLibrary` feature toggle